meson.build

```diff
@@ -223,6 +230,13 @@ gcab = find_program('gcab', required : true)
 bashcomp = dependency('bash-completion', required: false)
 python3 = find_program('python3')
 
+platform_deps = []
+if get_option('default_library') != 'static'
```

Comment:
I think that we might want to exclude compiling `default_library` of `static` on Windows build - set an error and abort meson.